src/editions/transitioning.md

```diff
@@ -1,11 +1,74 @@
 # Transitioning your code to a new edition
 
-In general, to update your code to a new edition, the process
-will look like this:
+Transitioning between editions is built around lints. Fundamentally, the
+process works like this:
 
-* Make sure your build is warning-free.
-* Update or add the `edition` key in `Cargo.toml`
-* Fix any warnings, possibly by running `rustfix`.
+* Get your code compiling with no warnings.
+* Opt in to the new edition.
+* Fix any new warnings that may result.
 
-At this time, the `edition` key does not yet work, and `rustfix` is in a
-larval state. This section will update as our story gets better here.
\ No newline at end of file
+Luckily, we've been working on a tool to help assist with this process,
+`rustfix`. It can take suggestions from the compiler and automatically
+re-write your code to comply with new features and idioms.
+
+> `rustfix` is still quite young, and very much a work in development. But it works
+> for the basics! We're working hard on making it better and more robust, but
+> please bear with us for now.
+
+## Installing rustfix
+
+You can get `rustfix` from crates.io. Given that you're probably using Cargo,
+you'll want to run this:
+
+```shell
+$ cargo +nightly install cargo-fix --git https://github.com/killercup/rustfix.git
+```
+
+And that's it! Note that you do need a nightly Cargo to use `rustfix` at the
```

Comment:
Technically you don't actually need a nightly Cargo for `rustfix`, it's just a nightly rustc currently. As an implementation detail though cargo-fix purely delegates to `cargo check`, which does end up transitively and effectively requiring nightly.

In any case though the `+nightly` on the `cargo install` in theory shouldn't be necessary even today!